rules/windows/process_creation/proc_creation_win_bitsadmin_download.yml

```diff
@@ -32,7 +32,7 @@ detection:
         CommandLine|contains: 'http'
     selection_copy:
         CommandLine|contains: 'copy bitsadmin.exe'
-    condition: (selection_img and selection_cmd) or (selection_img and all of selection_cli_*) or selection_copy
+    condition: selection_img and (selection_cmd or all of selection_cli_*) or selection_copy
```

Comment:
selection_copy is not a way to download a file.
I think we can remove it.

Reply:
Yeah. I left it because I don't know where to put it yet. But you're right. Let's remove it and add it later